CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # @digitalbazaar/cborld ChangeLog
 
+## 8.0.0 - 2024-mm-dd
+
+### Added
+- Add TypedLiteral, UntypedLiteral encoders and decoders to more generally
+  handle type-specific behavior such as cryptosuite string encoding as well
+  as pass through uncompressible untyped values.
+
+### CHANGED
+- Restructure term registry system to be more general with four tables which
```

Comment:
We need a note here that says that, by default, the behavior will be the same as in the previous minor version and that using these parameters is not allowed in `legacy` mode (we should also block these inputs as invalid API use if the `legacy` varint is being used).